src/content/posts/2023-03-27-leveling-up-your-project-testing-with-tsqlt-unit-tests-for-sql-queries.md

```diff
@@ -0,0 +1,115 @@
+---
+title: "Leveling Up Your Project Testing with tSQLt Unit Tests for SQL Queries"
+slug: leveling-up-your-project-testing-with-tSQLt-unit-tests-for-sql-queries
+date: 2023-03-27
+tags:
+- .NET
+- SQL
+- tSQLt
+- Unit Tests
+categories:
+- tSQLt
+twitter_text: "Leveling Up Your Project Testing with tSQLt Unit Tests for SQL Queries"
+authors: 
+- Cheng Yang
+image: https://images.unsplash.com/photo-1576444356170-66073046b1bc?ixlib=rb-4.0.3&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=1170&q=80
+image_url: https://unsplash.com/photos/EWLHA4T-mso
+image_credit: Ferenc Almasi
+---
+
+I am writing this blog post to share a cool unit testing tool that I learned about at the VS Live conference in Vegas. While many developers build tests for their code to make sure it is functioning as intended, fewer developers test their database queries. Testing your code and your database queries are equally vital to make sure your queries are working as intended and producing the expected results. In this blog article, I'll demonstrate how to use tSQLt, a unit testing framework for SQL Server, to create tests for your SQL Server database queries.
+
+### Step 1: Install tSQLt
+Go to the official website [tSQLt](https://tsqlt.org/downloads/) to download and set up tSQLt. Once the installation zip file has been downloaded, unzip it to a spot on your hard drive, and then run the `PrepareServer.sql` file. 
+<figure>
+<img src="/images/tsqlt/query1.png" style="max-width: 100%">
```

Comment:
```suggestion
<img src="/images/tsqlt/query1.PNG" style="max-width: 100%">
```

When I pulled your post down to check it out, I noticed that the images weren't loading for me - it seemed to be due to the casing on the file types.